packages/terra-modal/tests/jest/Modal.test.jsx

```diff
@@ -10,8 +10,3 @@ it('should mount an open modal', () => {
   const modal = mount(<ModalExample />);
   expect(modal).toMatchSnapshot();
 });
-
-it('should render an open modal', () => {
-  const modal = render(<ModalExample />);
```

Comment:
For those curious, this line produced the following error.

```
<--- Last few GCs --->

[42935:0x102801600]    25833 ms: Mark-sweep 1405.6 (1452.6) -> 1405.6 (1452.6) MB, 1810.7 / 0.0 ms  allocation failure GC in old space requested
[42935:0x102801600]    27430 ms: Mark-sweep 1405.6 (1452.6) -> 1405.6 (1440.6) MB, 1596.5 / 0.0 ms  last resort GC in old space requested
[42935:0x102801600]    29223 ms: Mark-sweep 1405.6 (1440.6) -> 1405.6 (1440.6) MB, 1792.8 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x123410ba5ee1 <JSObject>
    1: read [/Users/jl021230/Documents/PopulationHealth/terra/terra-core/node_modules/react-dom/cjs/react-dom-server.node.development.js:~2206] [pc=0x26d1330d7379](this=0x12341a902ef9 <ReactDOMServerRenderer map = 0x12342cf40529>,bytes=0x1234906829a9 <Number inf>)
    3: renderToStaticMarkup [/Users/jl021230/Documents/PopulationHealth/terra/terra-core/node_modules/react-dom/cjs/react-dom-server....

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
 1: node::Abort() [/Users/jl021230/.nvm/versions/node/v8.9.2/bin/node]
 2: node::FatalException(v8::Isolate*, v8::Local<v8::Value>, v8::Local<v8::Message>) [/Users/jl021230/.nvm/versions/node/v8.9.2/bin/node]
 3: v8::internal::V8::FatalProcessOutOfMemory(char const*, bool) [/Users/jl021230/.nvm/versions/node/v8.9.2/bin/node]
 4: v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationSpace) [/Users/jl021230/.nvm/versions/node/v8.9.2/bin/node]
 5: v8::internal::Runtime_AllocateInTargetSpace(int, v8::internal::Object**, v8::internal::Isolate*) [/Users/jl021230/.nvm/versions/node/v8.9.2/bin/node]
 6: 0x26d132f8463d
 7: 0x26d1330d7379
```

I posted on enzymes repo, but didn't get much luck in regards to a solution.